2020-12-01 13:48:12,430 INFO    MainThread:3947 [wandb_setup.py:_flush():70] setting env: {}
2020-12-01 13:48:12,430 INFO    MainThread:3947 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2020-12-01 13:48:12,430 INFO    MainThread:3947 [wandb_setup.py:_flush():70] setting login settings: {}
2020-12-01 13:48:12,430 INFO    MainThread:3947 [wandb_setup.py:_flush():70] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9f740b6fd0>
2020-12-01 13:48:12,430 INFO    MainThread:3947 [wandb_init.py:_log_setup():303] Logging user logs to /home/namitrak/Documents/Machine Learning/Image segmentation/wandb/run-20201201_134812-306bo03h/logs/debug.log
2020-12-01 13:48:12,431 INFO    MainThread:3947 [wandb_init.py:_log_setup():304] Logging internal logs to /home/namitrak/Documents/Machine Learning/Image segmentation/wandb/run-20201201_134812-306bo03h/logs/debug-internal.log
2020-12-01 13:48:13,025 INFO    MainThread:3947 [wandb_run.py:_console_start():1218] atexit reg
2020-12-01 13:48:13,025 INFO    MainThread:3947 [wandb_run.py:_redirect():1085] redirect: SettingsConsole.WRAP
2020-12-01 13:48:13,026 INFO    MainThread:3947 [wandb_run.py:_redirect():1116] Wrapping output streams.
2020-12-01 13:48:13,026 INFO    MainThread:3947 [wandb_run.py:_redirect():1132] Redirects installed.
2020-12-01 13:48:15,491 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:48:15,496 INFO    MainThread:3947 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:48:37,511 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:48:37,516 INFO    MainThread:3947 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:48:37,643 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:48:37,648 INFO    MainThread:3947 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:48:37,649 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:48:37,652 INFO    MainThread:3947 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:48:37,865 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:48:37,868 INFO    MainThread:3947 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:48:39,144 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:48:39,150 INFO    MainThread:3947 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:48:39,153 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:48:39,158 INFO    MainThread:3947 [wandb_init.py:_resume_backend():229] resuming backend
2020-12-01 13:48:39,161 INFO    MainThread:3947 [wandb_watch.py:watch():35] Watching
2020-12-01 13:48:40,447 INFO    MainThread:3947 [wandb_init.py:_pause_backend():224] pausing backend
2020-12-01 13:49:00,203 INFO    MainThread:3947 [wandb_run.py:_atexit_cleanup():1188] got exitcode: 0
2020-12-01 13:49:00,203 INFO    MainThread:3947 [wandb_run.py:_restore():1160] restore
2020-12-01 13:49:00,207 INFO    MainThread:3947 [wandb_run.py:_wait_for_finish():1305] got exit ret: uuid: "75d7c0b8019a455baffe7a266b53b002"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
      media_count: 1
    }
    pusher_stats {
      uploaded_bytes: 10064
      total_bytes: 10064
    }
  }
}

2020-12-01 13:49:02,211 INFO    MainThread:3947 [wandb_run.py:_wait_for_finish():1305] got exit ret: uuid: "780375a028e64534af4786d5e9a1b37e"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
      media_count: 1
    }
    pusher_stats {
      uploaded_bytes: 13540
      total_bytes: 13540
    }
  }
}

2020-12-01 13:49:03,281 INFO    MainThread:3947 [wandb_run.py:_show_summary():1450] rendering summary
2020-12-01 13:49:03,282 INFO    MainThread:3947 [wandb_run.py:_show_files():1517] logging synced files
2020-12-01 13:49:03,285 INFO    MainThread:3947 [internal.py:handle_exit():138] Internal process exited
